meilisearch/tests/index/test_index_document_meilisearch.py

```diff
@@ -18,6 +20,30 @@ def test_add_documents(empty_index, small_movies):
     assert index.get_primary_key() == 'id'
     assert update['status'] == 'processed'
 
+
+@pytest.mark.parametrize("batch_size", [2, 3, 1000])
+@pytest.mark.parametrize(
+    "primary_key, expected_primary_key", [("release_date", "release_date"), (None, "id")]
+)
+def test_add_documents_in_batches(
+    batch_size,
+    primary_key,
+    expected_primary_key,
+    empty_index,
+    small_movies,
+):
+    index = empty_index()
+    response = index.add_documents_in_batches(small_movies, batch_size, primary_key)
+    assert ceil(len(small_movies) / batch_size) == len(response)
+
+    for r in response:
+        assert "updateId" in r
+        update = index.wait_for_pending_update(r["updateId"])
+        assert update["status"] == "processed"
+
+    assert index.get_primary_key() == expected_primary_key
+
+
```

Comment:
```suggestion
```

Reply:
I have this extra space here to follow the [pep8 style guide](https://www.python.org/dev/peps/pep-0008/#blank-lines). However, I did this out of habit, and since the rest of the code doesn't follow this recommendation from pep8 I will remove it.